policies/aws_lambda_policies/aws_lambda_public_access.yml

```
@@ -0,0 +1,106 @@
+AnalysisType: policy
+Filename: aws_lambda_public_access.py
+PolicyID: "AWS.Lambda.PublicAccess"
+DisplayName: "AWS Lambda Public Access"
+Enabled: true
+ResourceTypes:
+  - AWS.Lambda.Function
+Tags:
+  - AWS
+  - Data Protection
+Reports:
+  MITRE ATT&CK:
+    - T1190
```

Nice MITRE tagging!  The Panther UI needs these in TA####:T#### format

```suggestion
    - TA0001:T1190
```